docs/index.md

```diff
@@ -379,6 +379,367 @@ def fig_to_html(fig: Figure) -> str:  # markdown-exec: hide
 print(fig_to_html(plt.gcf())) # markdown-exec: hide
 ```
 
+## Fitting a noisy sinusoid with quantum dropout
+
```

Comment:
hey @callumfduffy can you put this into `docs/dropout.md`  (rebase on main if you havent)